tools/mfassignr/mfassignr_mfassign.xml

```diff
@@ -19,6 +19,19 @@
         iso <- read.delim("$input_iso", sep="\t")
         #end if
         SN = $sn_ratio * $kmdn
+        POEx = 0
+        NOEx = 0
+
+        #if "$assign_typeof_ions"=="yes"
+            #if $ionMode=="pos"
+            POEx = 1
+            NOEx = 0
+            #end if
+            #if $ionMode=="neg"
+            POEx = 0
+            NOEx = 1
+            #end if
```

Comment:
Could be simplified with #if ... #else ... #end if

Reply:
resolved in [c7a2b4a](https://github.com/RECETOX/galaxytools/pull/595/commits/c7a2b4afa6af1acb99a77013ffd6dfb199a7c684)